citesphere-model/src/main/java/edu/asu/diging/citesphere/model/bib/impl/Citation.java

```diff
@@ -48,6 +48,7 @@
     private String series;
     private String seriesTitle;
     private String url;
+    private String note;
```

Comment:
what does this field contain? notes are individual items, aren't they?

Reply:
Note would be an individual item (Citation class instance). The `note` field contains the actual note String for the Note item type.

![image](https://user-images.githubusercontent.com/25862427/149559850-f20cce20-6889-4a54-8f39-1cc288b95688.png)
Although this response is from Zotero API, I have used the same field name for Citation class.

Reply:
Ah right.